packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -51,13 +51,15 @@
                 v-oc-tooltip="showSpaceMemberLabel"
                 :aria-label="showSpaceMemberLabel"
                 appearance="raw"
-                @click="openSidebarSharePanel(resource)"
+                @click="openSidebarSharePanel(resource as SpaceResource)"
               >
                 <oc-icon name="group" fill-type="line" />
               </oc-button>
             </template>
             <template #contextMenuActions="{ resource }">
-              <space-context-actions :action-options="{ resources: [resource] }" />
+              <space-context-actions
+                :action-options="{ resources: [resource] as SpaceResource[] }"
+              />
```

Comment:
I don't disagree, just curious: why was this necessary?

Reply:
My IDE complains that the types are incompatible: `Type Resource[] is not assignable to type SpaceResource[]`

Reply:
Yep, as discussed: makes sense and is correct (and so is this change)